tests/deployment/test_flow_runs.py

```
@@ -595,3 +596,335 @@ async def foo_flow() -> None:
+            idempotency_key="async-12345",
+            timeout=0,
+            poll_interval=0,
+            client=prefect_client,
+        )
+
+        assert flow_run_a.id == flow_run_b.id
+
+    async def test_arun_deployment_links_to_parent_flow(
+        self,
+        test_deployment: DeploymentResponse,
+        use_hosted_api_server,
```

Ruff (0.14.14)

Unused method argument: `use_hosted_api_server`

(ARG002)